tests/integrations/mcs/tso/server_test.go

```diff
@@ -578,3 +585,212 @@ func (suite *CommonTestSuite) TestBootstrapDefaultKeyspaceGroup() {
 	suite.pdLeader.ResignLeader()
```

Comment:
Please add a description for these three UTs. Or make the UT name more meaningful?